lib/SparqlQuery.js

```diff
@@ -1,13 +1,21 @@
+const MultiStream = require('multistream')
 const SparqlView = require('./SparqlView')
 
 class SparqlQuery extends SparqlView {
-  async _executeQuery (query, options, req, res) {
+  async _executeQuery (query, options) {
     const result = await this.client.constructQuery(query, options)
     if (!result.ok) {
       throw new Error(result.statusText)
     }
 
-    return res.graph(await result.quadStream())
+    return result.quadStream()
+  }
+
+  _sendResponse ({ queryResults, res }) {
+    const streams = queryResults.filter(qr => qr.readable)
```

Comment:
I'm not sure, but maybe it's better to use [isstream](https://www.npmjs.com/package/isstream). I thinke before the event `readable` was emitted, the property `readable` is false.

Reply:
ah, sure